internal/grpc/proto/kong/admin/model/v1/certificate.proto

```diff
@@ -37,4 +37,9 @@ message Certificate {
     (referenceable) = true
   ];
   repeated string tags = 8;
+  string subject = 9;
+  string issuer = 10;
+  string san = 11;
+  string expiry = 12;
+  string key_usage = 13;
```

Comment:
Can we actually move these fields into its own Protobuf message/entity, that way it can be re-used? I would just call it `CertificateMetadata`.